app/src/main/java/com/framgia/wsm/data/source/remote/api/service/WSMApi.java

```diff
@@ -3,25 +3,31 @@
 import com.framgia.wsm.data.model.Request;
 import com.framgia.wsm.data.model.RequestOff;
 import com.framgia.wsm.data.model.RequestOverTime;
+import com.framgia.wsm.data.model.User;
```

Comment:
ERROR: Unused import: com.framgia.wsm.data.model.User. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports